documentation/web/patterns/o-media--med-related.html

```diff
@@ -0,0 +1,49 @@
+---
+title: Media - Medium + Related Content
+tags: object
+date: 2016/08/06
+pattern_number: 1.1
+---
+<div class="example__container-lg">
+  <div class="o-media o-media--med">
+    <div class="o-media__content">
+      <h6 class="o-media__overline">Overline Ipsum</h6>
+      <h3 class="o-media__headline o-media__headline--h3">
+        <a class="o-media__headline-link" href="#">Headline ipsum sit amet dolor quando nonumy quaerend tortor</a>
+      </h3>
+      <div class="o-media__figure o-media__figure--widescreen">
+        <a href="#">
+          <img class="o-media__img" src="http://placehold.it/640x426">
+        </a>
+      </div>
+      <div class="o-media__teaser">Teaser ipsum sit amet dolor quando nonumy quaerend. In purus tortor, tincidunt consequat dolor vitae, commodo sagittis odio.</div>
+      <div class="o-media__meta">
+        <span class="o-media__label">
+          <svg class="icon icon--size-sm icon--left">
+            <use class="icon--line icon--color-primary" xlink:href="#icon_line-audio"/>
+          </svg>
+          Audio
+        </span>
+      </div>
+    </div>
+    <div class="o-media__related-content">
+      <h5 class="heading heading--h5 heading--uppercase u-text-color--gray">More in Arts and Entertainment</h6>
+      <ul class="c-list c-list--vert c-list--bullet">
+        <li class="o-media__teaser o-media__teaser--bullet">
+          <div class="o-media o-media--sm">
+            <div class="o-media__content">
+              <h4 class="o-media__headline o-media__headline--h5"><a href="#">Related headline ipsum sit amet dolor quando nonumy</a></h4>
```

Comment:
related link <a> tags should receive an `o-media__headline-link` class to give them our standard link hover underline effect